spec/controllers/state_file/intake_logins_controller_spec.rb

```diff
@@ -358,6 +359,34 @@
             expect(intake.reload.unfinished_intake_ids).to match_array([stub_intake.id.to_s])
           end
         end
+
+        context "when the intake has failed_attempt of 5" do
+          before do
+            intake.update(failed_attempts: 5, locked_at: locked_at)
+          end
+
+          context "if within lock out period" do
+            let(:locked_at) { 28.minutes.ago }
+
+            it "should redirect to lock out page" do
+              get :edit, params: params
+
+              expect(response).to redirect_to account_locked_intake_logins_path
```

Comment:
this happens due to a `before_action :redirect_locked_clients` in `ClientLoginsController` (which this controller inherits from)